jablib/src/main/java/org/jabref/logic/importer/fetcher/transformers/ScopusQueryTransformer.java

```diff
@@ -0,0 +1,103 @@
+package org.jabref.logic.importer.fetcher.transformers;
+
+import java.util.Optional;
+
+/**
+ * Query transformer for the Scopus Search API.
+ * <p>
+ * Scopus uses specific field codes for advanced search:
+ * - TITLE-ABS-KEY-AUTH() for searching in title, abstract, author and keywords
+ * - AUTH() for author search
+ * - SRCTITLE() for journal/source title search
+ * - PUBYEAR for publication year
+ * - DOI() for DOI search
+ * <p>
+ * Scopus supports AND, OR, AND NOT as boolean operators.
+ * <p>
+ * See: https://dev.elsevier.com/sc_search_tips.html
+ */
```

Comment:
Convert to markdown - or use proper HTML - the item list is not correctly displayed as is.